athina/evals/ragas/answer_relevancy/evaluator.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
+from ..ragas_evaluator import RagasEvaluator
+from athina.evals.eval_type import RagasEvalTypeId
+from athina.metrics.metric_type import MetricType
+from ragas.metrics import answer_relevancy
+
+
+class RagasAnswerRelevancy(RagasEvaluator):
+    """
+    This evaluator focuses on assessing how pertinent the generated response is to the given prompt. 
+    A lower score is assigned to responses that are incomplete or contain redundant information.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
You can remove this. 

Python will automatically call the constructor of the superclass if you don't provide a constructor in the subclass.

Reply:
Thanks. Fixed